### PR TITLE
Esltiming 50

### PR DESCRIPTION
**Pull request by kukhee**
Resolve firmware reprogramming issue in linuxRT. 
C++ compiler in linuxRT is more picky, the change makes compiler happy to work.

### CODE REVIEW

**Comment by kukhee** [comment]
We have tested it in B84 test stand (cpu-b084-test2) with C02 (legacy PROM) card and  C03 (new PROM) card.
Minh has tested  it in B34 test stand (cpu-b34-test3 and cpu-b34-test4) with both C02 and C03.

File: src/McsRead.cpp [hunk 1]
```diff
@@ -111,7 +111,7 @@ uint32_t McsRead::addrSize ( ) {
 
 // Get next memory data and address index
 int32_t McsRead::read (McsReadData *mem) {
-   int32_t status = 0;
+   int32_t status;
```
Comment (JJL772) [inline src/McsRead.cpp:114]: `status` may be used uninitialized if `endOfFile` is true before entering the loop below. That condition is probably unlikely though

**Review by JJL772 — APPROVED** [review]

File: src/McsRead.cpp (hunk 1 above)
Comment (kukhee) [reply src/McsRead.cpp:114]: I agree, Thank you.

Comment (ruck314) [reply src/McsRead.cpp:114]: @JJL772 & @kukhee : Can we migrate these changes back to the original source code that this was copied from?
```
-   int32_t status = 0;
+   int32_t status;
```
https://github.com/slaclab/evr-card-g2/blob/master/software/PromLoader/McsRead.cpp

Comment (kukhee) [reply src/McsRead.cpp:114]: Hi Larry, Thank you for your advice. I will retake test shortly then, will migrate it back to the original source in the evr-card-g2. Thank you.

**Pull request merged**